doc: add Google Group instructions to onboarding (#1192)
nodejs/TSC: ONBOARDING.md

* Invite them to the HackerOne [Node.js team](https://hackerone.com/nodejs/team_members).
  Have them turn on two-factor authentication after they have joined.
* Add them to the `tsc` and `crypto-export` mailing lists.

> * Add them to the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group as owners, so they receive Calendar invites for TSC meetings.

Reviewer: Should I be able to access this link? I get a "content unavailable" error message.

Reviewer: Ah never mind. Just saw the invite and the link works after accepting.

* Remove them from the `@pkgjs` org unless they are members for a
  reason other than TSC membership.
* Remove them from the `tsc` and `crypto-export` mailing lists.

> * Add them to the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group so they receive Calendar invites for TSC meetings.

Reviewer 2: ```suggestion
* Remove them from the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group.
```

Reviewer 3: Good catch!
